richardltyler/vueniverse
# Fix home route rendering

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
Make PicOfTheDay the Home route
Add styling to PicOfTheDay


## *Instructions for reviewer:*
*Where should the reviewer start?*
index.js line 2

*How should this be manually tested?*
Should be able to go to the About page and not see the PicOfTheDay component, then back to Home and see the PicOfTheDay

## *Issues:*
Need to conditional render for the copyright key in the fetch object



## Checklist:
